tm2/pkg/bft/state/store.go

```diff
@@ -175,7 +180,7 @@ func LoadABCIResponses(db dbm.DB, height int64) (*ABCIResponses, error) {
 // SaveABCIResponses persists the ABCIResponses to the database.
 // This is useful in case we crash after app.Commit and before s.Save().
 // Responses are indexed by height so they can also be loaded later to produce Merkle proofs.
-func saveABCIResponses(db dbm.DB, height int64, abciResponses *ABCIResponses) {
+func SaveABCIResponses(db dbm.DB, height int64, abciResponses *ABCIResponses) {
```

Comment:
```suggestion
// NOTE: this should only be used internally by the bft package and subpackages.
func SaveABCIResponses(db dbm.DB, height int64, abciResponses *ABCIResponses) {
```

Reply:
I'd suggest moving it in an internal/ package, but it looks complex to do, so let's roll with this for now.

Reply:
applied in [e649de5](https://github.com/gnolang/gno/pull/1941/commits/e649de519ea1fd7ec60399c7ab4df0b85c7e8cde)